lib/slack_announcer.rb

```diff
@@ -33,4 +34,11 @@ def dashboard_host_name
       "staging" => "grafana.staging.publishing.service.gov.uk",
     }[@environment_name]
   end
+
+  def dashboard_url(host_name, application_name)
+    url = "https://#{host_name}/api/dashboards/file/deployment_#{application_name}.json"
+    return nil unless HTTP.get(url).code == 200
```

Comment:
Not that we know of, but it seems worth changing this to work for any 2xx/3xx responses. We've changed this to use `range`.